### PR TITLE
Generates shell orbitals

### LINKED ISSUE #32: Periodic Table shell orbitals
- [x] s2...d4 orbitals should be s1...d5
- [x] s2...d9 orbitals should be s1...d10

### PR DESCRIPTION
**Pull request by tyfoster97**
Closes #32

### CODE REVIEW

File: src/periodic-table/Element.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  "5f",
+  "6d",
+  "7p",
+];
+
+interface ElectronShells {
+  [shell: string]: Shell;
+}
+
+export const getShells = (e: Element, charge = 0) => {
```
Comment (timstoddard) [inline src/periodic-table/Element.ts:114]: can you please add some unit tests for this logic? I'm not so familiar with orbitals so I'd like to see some true/false/edge cases. please use jest.

**Review by timstoddard — COMMENTED** [review]
also why was the package lock changed?
